### PR TITLE
update: oraclelinux9 support

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area crawler

**What this PR does / why we need it**:

Updated oracle to support OracleLinux9.
Moreover, unified OracleLinux{6,7,8,9} under single `OracleLinux`.
Finally, renamed output target to `ol`, that is the `ID` got from `/etc/os-release` on OracleLinux.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: kernel_crawler/oracle.py [hunk 1]
```diff
@@ -1,70 +1,46 @@
...
 
     def list_repos(self):
         return [OracleRepository(url) for url in self.repos()]
 
     def to_driverkit_config(self, release, deps):
         for dep in deps:
             if dep.find("devel") != -1:
-                return repo.DriverKitConfig(release, "oracle8", dep)
\ No newline at end of file
+                return repo.DriverKitConfig(release, "ol", dep)
```
Comment (FedeDP) [inline kernel_crawler/oracle.py:46]: @EXONER4TED this change should be ported also to Driverkit! 
Can we do it in https://github.com/falcosecurity/driverkit/pull/260?

Comment (EXONER4TED) [reply kernel_crawler/oracle.py:46]: On it!

**Review by maxgio92 — APPROVED** [review]
/approve

Cool @FedeDP!

**Pull request merged**